docs/multicluster/architecture.md

```diff
@@ -34,33 +36,42 @@ given ClusterSet.
 
 ## Antrea Multi-cluster Controller
 
-In a member cluster, Antrea Multi-cluster creates a Deployment that runs Antrea Multi-cluster
-Controller which is responsible for exporting resource to and importing resource from a leader
-cluster in a ClusterSet.
+Antrea Multi-cluster Controller watches ClusterSet, ClutserClaim, Service, ServiceExport etc.
+It has different responsibilities in member and leader clusters.
```

Comment:
In either a leader or a member cluster, Antrea Multi-cluster Controller is deployed with a Deployment of a single replica, but it takes different responsibilities in member and leader clusters.